changelogs/fragments/integration_tests_fixes.yml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
```

Comment:
Can you change these to `trivial`?  Internal testing changes are unnecessary to show in the release changelogs, and `trivial` is basically the only way to suppress them from the scripts.